HomeToGather/HomeToGather/View/TicketList/Invited/CardView.swift

```diff
@@ -120,6 +104,36 @@ struct BackgroundClearView: UIViewRepresentable {
     func updateUIView(_ uiView: UIView, context: Context) {}
 }
 
+struct FocusView: View {
+    @FocusState private var isFocused: Bool
+    @State private var feedback: String = ""
```

Comment:
```suggestion
    @State private var feedback = ""
```

Swift에서 명확한 타입은 타입표기를 생략할 수 있어요 : )

Reply:
그럼 혹시 코드를 작성할 때 명확한 타입 선언은 지양하는 편이 좋을까요?

Reply:
네 알겠습니다 👍